_test/test_utilities/test_skip.m

```diff
@@ -0,0 +1,22 @@
+classdef test_skip < TestCase
+    methods
+        function this=test_skip(varargin)
+            if nargin>0
+                name = varargin{1};
+            else
+                name  = 'test_skip';
+            end
+            this = this@TestCase(name);
+
+        end
+
+        %------------------------------------------------------------------
+        function test_skip_with_message(this)
+            skipTest('This test is skipped because of custom reasons.');
```

Comment:
Would there be value in adding some code after the `skipTest` command to prove that it aborts execution at that line? e.g. 
```
disp("skipTest failed to halt execution"); 
assertFalse(true);
```